config/services.yml

```diff
@@ -0,0 +1,70 @@
+---
+parameters:
+  env(ACLI_DATA_DIR): "%env(HOME)%"
+  app.data_dir: "%env(HOME)%/.acquia"
+  app.ssh_dir: "%env(HOME)%/.ssh"
+  app.acli_config_filename: 'acquia-cli.json'
+  app.cloud_config_filename: 'cloud_api.conf'
+  app.acli_config_filepath: "%app.data_dir%/%app.acli_config_filename%"
+  app.cloud_config_filepath: "%app.data_dir%/%app.cloud_config_filename%"
+
+services:
+
+  _defaults:
+    autoconfigure: true
+    autowire: true
+    bind:
+      $cloudConfigFilepath: '%app.cloud_config_filepath%'
+      $acliConfigFilename: '%app.acli_config_filename%'
+      $repoRoot: "%kernel.project_dir%"
+      $sshDir: "%app.ssh_dir%"
+      Webmozart\KeyValueStore\JsonFileStore $datastoreCloud: '@datastore.cloud'
+      Webmozart\KeyValueStore\JsonFileStore $datastoreAcli: '@datastore.acli'
+    public: true
+
+  # Register nearly all Acquia CLI classes as services.
```

Comment:
Why would a command be a service? Is that required for injecting a dependency into it?

Reply:
In Symfony 5, every command is a service. It's required for dependency injection.

> If you're using the default services.yaml configuration, your command classes are already registered as services. Great! This is the recommended setup.


https://symfony.com/doc/current/console/commands_as_services.html